src/test/java/uk/gov/hmcts/ccd/domain/service/startevent/DefaultStartEventOperationTest.java

```diff
@@ -128,6 +131,9 @@
                                                      + "}");
             data.put("OrganisationPolicyField", orgPolicy);
 
+            JsonNode textFieldDefaultValue = MAPPER.readTree("{ \"TextField0\": null }");
+            data.put("TextField0", textFieldDefaultValue);
```

Comment:
May also be worth adding another top level field in the `existingCaseData` with a non-null value, with a non-null value also in the default, to verify that it doesn't default when it shouldn't.